server/graphql/common/expenses.ts

```diff
@@ -1172,6 +1197,17 @@ const checkTaxes = (account, host, expenseType: string, taxes): void => {
   }
 };
 
+/**
+ * Throws if the accounting category is not allowed for this expense/host
+ */
+const checkCanUseAccountingCategory = (accountingCategory: AccountingCategory, host: Collective): void => {
```

Comment:
If we have a safe return for an empty `accountCategory` parameter:
```suggestion
const checkCanUseAccountingCategory = (accountingCategory?: AccountingCategory, host: Collective): void => {
```

Reply:
> error TS1016: A required parameter cannot follow an optional parameter.

Reply:
Problem is: we don't want devs to call this function without providing the parameter. So the right call is to type it as `AccountingCategory | undefined | null`. Since we're not enforcing strict null checks for now, I don't think that we should bother with that, but I've done it here since I was on it